modules/crud.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-unused-vars */
+const containerElement = document.querySelector('.containerTodo');
+const ulElement = document.querySelector('.ulElement');
+
+const Task = class {
+  constructor(description, completed = false, index = 0) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+    this.tasks = [
+      {
+        description: 'wash the dogs',
+        completed: false,
+        index: 0,
+      },
+      {
+        description: 'Complete To Do list project',
+        completed: false,
+        index: 1,
+      },
+    ];
```

Comment:
- [Optional] Great job for implementing the add, remove and complete tasks functionality 👍👍 when you add new tasks you are adding additional two tasks of the objects inside the new object because you initialize the constructor with additional tasks object, please remove the `line 10-21` to have clear object 👌